src/constants/abis.ts

```diff
@@ -0,0 +1,13 @@
+export const INBOX_ABI = [
```

Comment:
Is this for Arbitrum or some other chains?

Reply:
Opps just old file forgot to remove